ibis/backends/tests/test_vectorized_udf.py

```diff
@@ -14,33 +14,100 @@
 pytestmark = pytest.mark.udf
 
 
-@elementwise(input_type=[dt.double], output_type=dt.double)
+# elementwise UDF
 def add_one(s):
     assert isinstance(s, pd.Series)
     return s + 1
 
 
-@analytic(input_type=[dt.double], output_type=dt.double)
+def create_add_one_udf(result_formatter):
+    return elementwise(input_type=[dt.double], output_type=dt.double)(
+        _format_udf_return_type(add_one, result_formatter)
+    )
+
+
+def _format_udf_return_type(func, result_formatter):
+    def _wrapper(*args, **kwargs):
```

Comment:
can you move near the top as this is pretty general & add a doc-string (just to describe how to use is enough)